gammapy/image/core.py

```diff
@@ -53,15 +44,41 @@ class SkyImage(object):
     meta : `~collections.OrderedDict`
         Dictionary to store meta data.
     """
+    AxisIndex = namedtuple('AxisIndex', ['x', 'y'])
+    ax_idx = AxisIndex(x=1, y=0)
```

Comment:
@OlgaVorokh @adonath - FYI: I introduced this as a helper to get the axis order right. (It was incorrect in `SkyImage.center`, fixed in this PR.)
I think we should also expose something like this publicly, but I wasn't sure exactly what (`ax_idx`, `naxis`?) so for now I just use this internally.